.github/infrastructure/terraform/conformance/pubsub/aws/snssqs/snssqs.tf

```diff
@@ -0,0 +1,94 @@
+resource "aws_sns_topic" "testTopic" {
+  name = "testTopic"
+  tags = {
+    dapr-topic-name = "testTopic"
```

Comment:
Suggestion - Specifying the values for **name** and **dapr-topic-name** as a Terraform variable with default values would allow modification at runtime.
Something like:
```terraform
#variables.tf
variable "test_topic" {
  type            = string
  description = "The Pub/Sub topic name."
  default        = "testTopic"
}

#snssqs.tf
 name = var.test_topic
 tags = {
   dapr-topic-name = var.test_topic
}


```